utils_nlp/models/transformers/extractive_summarization.py

```diff
@@ -96,12 +172,12 @@ def __init__(self, file_list, is_shuffle=False):
                 files is shuffled when the dataset is loaded. Defaults to False.
         """
 
-        self.file_list = file_list
+        self.file_list = sorted(file_list)
```

Comment:
Is sorting really needed here? Might conflict with users' expectation in case the list is provided in a certain order.

Reply:
yes, it's required. I have observed if it's not sorted, the evaluation result through AML is very bad.